playbook.yml

```diff
@@ -50,6 +50,9 @@
 
     - include_role:
         name: "fio-distributed"
+      loop: "{{ range(0, (workload.args.samples | int)) | list | flatten(1)}}"
```

Comment:
I don't think the loop control belongs in the `playbook.yaml`. We've used a different convention with at least uperf and pgbench, leveraging jinja2 loops in the template. I think the same thing can be applied to fio-d in the client.yaml file.

Also... Wouldn't looping here result in a separate UUID generated for each loop iteration and mess up the indexing?